scripts/make-env

```diff
@@ -94,4 +94,6 @@ export OPENBALENA_REGISTRY_SECRET_KEY=$(randstr 32)
 export OPENBALENA_SSH_AUTHORIZED_KEYS=
 export OPENBALENA_SUPERUSER_EMAIL=$SUPERUSER_EMAIL
 export OPENBALENA_SUPERUSER_PASSWORD=$(printf "%q" "${SUPERUSER_PASSWORD}")
+export OPENBALENA_USE_ACME_CERT=false
+export OPENBALENA_USE_ACME_PRODUCTION_CERT=false
```

Comment:
I think we’ll soon need to surface these further given how most people seem to be using openBalena now and how easy this makes working with certs. Can you add a quickstart optional flag to turn these on?

Reply:
Sure. Shouldn't be too much work.